keyboards/moondrop/dash75/r1/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 moondroplab (@moondroplab)
```

Comment:
is the r1 revision actually needed?   Eg, is there a planned r2, etc?

Reply:
yes - the next variant is in development